Samantha-samanthalpark/Homework-1/99-bottles-of-beer/app.js

```diff
@@ -0,0 +1,21 @@
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal.
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else).
+*/
+let bPlural = " bottles of beer";
+let bSingle = " bottle of beer";
+
+for(let i = 100; i > 0; --i){
```

Comment:
It's best practice to include a space after keywords like `for`, and both before and after words like `else`. This won't change how your code is processed by a browser, but it makes it easier for other developers to read and understand.